test/liquidity-curve.test.js

```diff
@@ -105,91 +119,6 @@ describe('LiquidityCurve', function () {
   })
 
   describe('combine', function () {
-    it('doesnt create non-increasing y coordinates', function () {
```

Comment:
Can't we convert these tests to the new data format and keep them, instead of removing them? It took @momerath42 and me a lot of work to find these edge case examples :)